server/routes/createGoal/CreateGoalController.ts

```diff
@@ -50,6 +50,9 @@ export default class CreateGoalController {
     const selectedAreaOfNeed = areasOfNeed.find(areaOfNeed => areaOfNeed.url === req.params.areaOfNeed)
     const minimumDatePickerDate = formatDateWithStyle(new Date().toISOString(), 'short')
 
+    // TODO: How many places can I get here from?
```

Comment:
not quite sure what these TODOs are meant to be suggesting. If there's something concrete here TODO in the future, can you be specific?

Reply:
Removed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/186/commits/2a84f8fc79e24634664948ecb231cd94aa9de143